polaris.shopify.com/next.config.js

```diff
@@ -397,11 +397,12 @@ const layoutAndStructure = [
     destination: '/components/layout-and-structure/grid',
     permanent: false,
   },
-  {
-    source: '/components/inline',
-    destination: '/components/layout-and-structure/horizontal-stack',
-    permanent: false,
-  },
+  // TODO: Re-enable this redirect once the new inline stack component is released
+  // {
+  //   source: '/components/layout-and-structure/horizontal-stack',
+  //   destination: '/components/layout-and-structure/inline-stack',
+  //   permanent: false,
+  // },
```

Comment:
could you not do this now if the doc updates will only be published once we merge in `next` and release v12?

Reply:
Not sure why but this was causing build errors. The sitemap failed thinking this was still a valid route even though it's only supposed to be a redirect. I'll dig in further when this is closer to shipping. I don't want to disrupt other se23 workflows